pkg/windows/windows.go

```diff
@@ -879,8 +885,9 @@ func (vm *windows) ensureHNSNetworksAreRemoved() error {
 // removeHNSNetwork removes the given HNS network.
 func (vm *windows) removeHNSNetwork(networkName string) error {
 	cmd := getHNSNetworkCmd(networkName) + " | Remove-HnsNetwork;"
-	// PowerShell returns error waiting without exit status or signal error when the OVNKubeOverlayNetwork is removed.
-	if out, err := vm.Run(cmd, true); err != nil && !(networkName == OVNKubeOverlayNetwork && strings.Contains(err.Error(), cmdExitNoStatus)) {
+	// PowerShell returns error waiting without exit status or signal error when the networks are removed.
+	out, err := vm.Run(cmd, true)
+	if err != nil && !strings.Contains(err.Error(), cmdExitNoStatus) {
```

Comment:
This change needs to be added to commit msg, will update in next iteration.
Removing both networks has been leading to error waiting without exit status, in my opinion we can ignore error for both network types as we have a poll with a check for hnsNetwork call following this method in ensureHNSNetworksAreRemoved().

Reply:
No I don't think this change is required I will remove it in my next iteration.
Apologies for some unrelated changes left over for debugging.